its/plugin/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id("org.sonarsource.iac.code-style-convention")
+    id("org.sonarsource.iac.java-conventions")
+    id("org.sonarsource.iac.integration-test")
+}
+
+description = "SonarSource IaC Analyzer :: Integration Testing"
+
+dependencies {
+    "integrationTestImplementation"(project(":sonar-iac-plugin", configuration = "shadow"))
+    "integrationTestImplementation"(libs.junit.jupiter)
+    "integrationTestImplementation"(libs.assertj.core)
+    "integrationTestImplementation"(libs.sonar.orchestrator.junit5)
+    "integrationTestImplementation"(libs.sonar.ws)
+    "integrationTestImplementation"(libs.sonar.lint)
+    "integrationTestImplementation"(libs.sonar.plugin.api)
+}
+
+tasks.integrationTest {
+    inputs.files("../../sonar-iac-plugin/build/libs")
```

Comment:
Shouldn't be needed as you already have a dependency on sonar-iac-plugin artifact from `shadow` configuration